proto/service.go

```diff
@@ -16,7 +16,7 @@ type UniqueInstance struct {
 
 type ServiceInstance struct {
 	Service    string // internal (agent, log, data) or and ExternalService
-	InstanceId uint   // external service instance ID: server-5, mysql-9, etc.
+	InstanceId string // external service instance UUID.
```

Comment:
I saw the contradiction, specs says one thing, your code another. But in any case, I don't think this should even exist in PCTv3 (?) . The change was primarily to be able to compile the thing and run instance tests until they pass.